components/places/android/src/test/java/mozilla/appservices/places/PlacesConnectionTest.kt

```diff
@@ -359,25 +361,25 @@ class PlacesConnectionTest {
             title = "example folder"
         )
 
-        val item = db.getBookmark(itemGUID)!! as BookmarkItem
-        val sep = db.getBookmark(sepGUID)!! as BookmarkSeparator
-        val folder = db.getBookmark(folderGUID)!! as BookmarkFolder
+        val item = db.getBookmark(itemGUID)!! as ReadonlyBookmarkItem.Bookmark
+        val sep = db.getBookmark(sepGUID)!! as ReadonlyBookmarkItem.Separator
+        val folder = db.getBookmark(folderGUID)!! as ReadonlyBookmarkItem.Folder
 
-        assertEquals(item.type, BookmarkType.Bookmark)
-        assertEquals(sep.type, BookmarkType.Separator)
-        assertEquals(folder.type, BookmarkType.Folder)
+        // assertEquals(item.type, BookmarkType.BOOKMARK)
```

Comment:
I think it's fine to delete these checks - even in the old version they seem a bit pointless - we've explicitly cast just a few lines up (ie, it is was possible to cast a Separator as a Folder, I suspect the type would be Folder.)